mozci/console/commands/push.py

```diff
@@ -787,8 +877,8 @@ def handle(self) -> None:
                     )
 
             if push.backedout or push.bustage_fixed_by:
-                group_classifications: dict[
-                    str, dict[tuple[str, str], set[str]]
+                group_classifications: Dict[
+                    str, Dict[Tuple[str, str], Set[str]]
```

Comment:
Nit: why did you move to typing instead of built-in names?

Reply:
Linting was failing:
```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

mozci/console/commands/push.py:880: error: "dict" is not subscriptable, use "typing.Dict" instead
mozci/console/commands/push.py:880: error: "tuple" is not subscriptable, use "typing.Tuple" instead
mozci/console/commands/push.py:880: error: "set" is not subscriptable, use "typing.Set" instead
Found 3 errors in 1 file (checked 39 source files)
```

The change was introduced by https://github.com/mozilla/mozci/pull/781 and merged with a failing pipeline

Reply:
Oh no, I did notice, it was fixed in #784.

Reply:
So it might be because I'm using python 3.8, do you want me to revert the change?

Reply:
Python 3.10 doesn't like me ahah:
> The virtual environment found in /home/eva/.cache/pypoetry/virtualenvs/mozci-BCG4KwUI-py3.10 seems to be broken.

Also you are using typing everywhere in mozci not built-in names

Reply:
Yeah we were using typing because the feature to use built-in names was not available yet, but it is the future